scripts/build.ps1

```diff
@@ -83,21 +86,21 @@ if (!(Test-Path -Path "$OutputPath\build\native\include\jsi")) {
 if (!(Test-Path -Path "$OutputPath\license")) {
     New-Item -ItemType "directory" -Path "$OutputPath\license" | Out-Null
 }
-if (!(Test-Path -Path "$OutputPath\lib\$Configuration\$Platform")) {
-    New-Item -ItemType "directory" -Path "$OutputPath\lib\$Configuration\$Platform" | Out-Null
+if (!(Test-Path -Path "$OutputPath\lib\$Architecture\$Configuration\$Platform")) {
+    New-Item -ItemType "directory" -Path "$OutputPath\lib\$Architecture\$Configuration\$Platform" | Out-Null
 }
 
 # Binaries
 if (!$PSVersionTable.Platform -or $IsWindows) {
-    Copy-Item "$buildoutput\v8jsi.dll" -Destination "$OutputPath\lib\$Configuration\$Platform"
-    Copy-Item "$buildoutput\v8jsi.dll.lib" -Destination "$OutputPath\lib\$Configuration\$Platform"
-    Copy-Item "$buildoutput\v8jsi.dll.pdb" -Destination "$OutputPath\lib\$Configuration\$Platform"
+    Copy-Item "$buildoutput\v8jsi.dll" -Destination "$OutputPath\lib\$Architecture\$Configuration\$Platform"

```

Comment:
Out of curiosity, shouldn't MSBuild naturally take care of this as longa s `IntDir` and `OutDir` are correctly set? (See RNW's `Directory.Build.props`).

Reply:
V8 doesn't use MSBuild, it uses proprietary build tools built on top of GN

---
In reply to: [468251030](https://github.com/microsoft/v8-jsi/pull/20#discussion_r468251030) [](ancestors = 468251030)